libraries/system.rb

```diff
@@ -3,12 +3,8 @@ module System
     class FormFactor
       attr_reader :machine_model
 
-      def initialize(machine_model = nil)
-        @machine_model = if Chef.node.nil?
```

Comment:
I know these were all deleted, but you could use a ternary for a one-line if/else:

@machine_model = Chef.node.nil? ? machine_model : Chef.node['hardware']['machine_model']

Reply:
Indeed!